build.sbt

```diff
@@ -102,15 +102,20 @@ lazy val semanticdbScalacCore = project.in(file("semanticdb/scalac/library")).se
   sharedSettings,
   publishJVMSettings,
   fullCrossVersionSettings,
-  protobufSettings,
   mimaPreviousArtifacts := Set.empty,
   moduleName := "semanticdb-scalac-core",
   description := "Library to generate SemanticDB from Scalac 2.x internal data structures",
-  Compile / unmanagedSourceDirectories ++= {
-    val base = (ThisBuild / baseDirectory).value / "semanticdb"
-    List(base / "metap", base / "cli", base / "semanticdb", base / "metacp", base / "symtab")
-  },
   libraryDependencies += "org.scala-lang" % "scala-compiler" % scalaVersion.value
+).dependsOn(semanticdbShared)
+
+lazy val semanticdbShared = project.in(file("semanticdb/semanticdb")).settings(
+  moduleName := "semanticdb-shared",
```

Comment:
this is missing on other projects, I don't know if it's intentional:
> [info] 	published semanticdbmetacp_2.13.14 to https://oss.sonatype.org/content/repositories/snapshots/org/scalameta/semanticdbmetacp_2.13.14/4.9.7+12-060037a8-SNAPSHOT/semanticdbmetacp_2.13.14-4.9.7+12-060037a8-SNAPSHOT.jar

https://github.com/scalameta/scalameta/actions/runs/9646040618/job/26604944229

Reply:
i will fix that. not intentional. i assumed the name implicitly came from the subdir but it seems it comes from the sbt variable name (quite unexpected). thank you for alerting me to this.